README.md

```diff
@@ -193,6 +193,38 @@ like Android where it's hard to set the system library path), you can do so usin
 * glib: `vipsffm.libpath.glib.override`
 * gobject: `vipsffm.libpath.gobject.override`
 
+## Memory Usage
+
+In long-running (think server) applications you may find libvips using a lot of memory, this is probably caused by two
+things:
+
+### The Operation Cache
+
+By default libvips will cache the 100 most recent image operations, if you're processing lots of different images you
+will probably want to disable it:
+
+```java
+Vips.init();
+VipsRaw.vips_cache_set_max(0);
```

Comment:
I'm not sure this is the best way to do this - I think the use of VipsRaw is discouraged (but it does seem to work).

Reply:
I think this could probably be lifted to the Vips class if it's commonly used by server systems

Reply:
I added `Vips.disableOperationCache` as a helper function for this.